debug_toolbar/panels/sql/tracking.py

```diff
@@ -88,7 +88,12 @@ def __init__(self, cursor, db, logger):
 
     def _quote_expr(self, element):
         if isinstance(element, six.string_types):
-            return "'%s'" % force_text(element).replace("'", "''")
+            try:
+                return "'%s'" % force_text(element).replace("'", "''")
+            except DjangoUnicodeDecodeError:
+                return repr(element)
+        elif isinstance(element, six.binary_type):
+            return "(binary data)"
```

Comment:
Should this use `repr(element)` as well? I guess I'm confused why sometimes binary data is represented but other times it is not. Or alternatively, should above be:

```py
            except DjangoUnicodeDecodeError:
                return "(binary data)"
```

Is this branch tested? I don't see `(binary data)` added to the tests.

Reply:
I agree, it should be `return repr(element)` but in that case there is no point specifically testing for `elif isinstance(element, six.binary_type):` because the `else:` clause also returns `return repr(element)`.